Dockerfile.ocp

```diff
@@ -13,20 +13,37 @@ RUN if [ $(uname -m) = "x86_64" ]; then \
       touch /tmp/esp.img; \
     fi
 
+## Build iPXE w/ IPv6 Support
+## Note: we are pinning to a specific commit for reproducible builds.
+## Updated as needed.
+RUN git clone http://git.ipxe.org/ipxe.git && \
```

Comment:
I'm pretty sure this is not accessible downstream :/

Reply:
It's accessible, although certainly not ideal. We'll have the right things in RPM's soon(ish) https://github.com/openshift/ironic-image/pull/45#discussion_r365261838